styles.css

```diff
@@ -0,0 +1,156 @@
+body {
+  background: url(images/Headerbg.svg) no-repeat;
+  background-size: auto;
```

Comment:
- [ ] The background image is not properly placed.

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/121397233-0eec5980-c94c-11eb-886b-6b58c7f7b73a.png) | ![image](https://user-images.githubusercontent.com/16058470/121397811-989c2700-c94c-11eb-8bc1-d08e84484ffb.png)